cosmos/errors.go

```diff
@@ -0,0 +1,14 @@
+package cosmos
+
+import "github.com/cosmos/cosmos-sdk/types"
+
+const (
+	CodeForbidden types.CodeType = 403
+	CodeNotFound                 = 404
+	CodeExist                    = 409
+)
+
+const (
+	CodespaceDatabase types.CodespaceType = "database"
+	CodespaceAccount                      = "account"
```

Comment:
what about only one codespace? cosmos seems to use only one:
https://github.com/cosmos/cosmos-sdk/blob/d81d46192a0c26a4ed8bec911813b3239c14a55e/types/errors.go#L56-L57

Reply:
yes cosmos is using one, but left it configurable, so we can adjust the errors.